db/seeds.rb

```diff
@@ -5,3 +5,38 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+def create_profile(uid:, given_name:, surname:,  department:, supervisor: nil, biweekly: false)
+    user = User.create!(uid: uid)
+    StaffProfile.create!(given_name: given_name, surname: surname, 
+                         department: department, biweekly: biweekly,
+                         user: user, email: "#{user.uid}@princeton.edu")
+end    
+
+oul = Department.create!(name: 'Library-Office of the University Librarian')
+anne_profile = create_profile(uid: 'ajarvis', department: oul,
+                              given_name: 'Anne', surname: 'Jarvis')
+
+lit = Department.create!(name: 'Library Information Technology', head: anne_profile)
```

Comment:
I'm not sure this is a department? Jon is a member of ITIMS, right?

Reply:
You are correct.  I was swapping between the Library Staff search and the Princeton one and got confused.